xep-0077.xml

```diff
@@ -517,9 +524,12 @@
   </ol>
   <p>Support for extensibility via <cite>Data Forms</cite> is RECOMMENDED but is not required for compliance with this document.</p>
 </section1>
-<section1 topic='Redirection' anchor='redirect'>
-  <p>A given deployment MAY wish to redirect users to another medium (e.g., a website) for registration, rather than allowing in-band registration. The recommended approach is to include only the &lt;instructions/&gt; element rather than the required fields or a data form in the IQ result, as well as a URL encoded using &xep0066; (see the <link url="#precedence">Precedence Order</link> section below for further details).</p>
-  <example caption='Host Redirects Entity to Web Registration'><![CDATA[
+<section1 topic='Alternative Registration Medium' anchor='alternative-registration-medium'>
+  <p>A given deployment MAY wish to provide an alternative medium (e.g., a website) for registration.</p>
+  <section2 topic='Web Registration' anchor='web-registration'>
```

Comment:
This invalidates prior links to this subsection (https://xmpp.org/extensions/xep-0077.html#redirect). I propose we keep that URL as is (by keeping `anchor='redirect'`) as is. But this is probably editorial and thus is on the editor to decide.

Reply:
I thought about that as well when I changed the anchor but keeping it would result in a misleading anchor. Is there any solution for aliases/redirections from old URLs to new ones even if they do not exactly match?

Reply:
@Kev What should I do?